scheduler/src/cook/mesos/scheduler.clj

```diff
@@ -595,7 +594,7 @@
                                         (sort-by (comp :db/id :job #(.getRequest %))))
         :let [request (.getRequest task)
               task-id (:task-id request)
-              job-id (get-in request [:job :db/id])]]
+              job-id [:job/uuid (get-in request [:job :job/uuid])]]]
```

Comment:
Should the variable name be updated, it is no longer an `id`, it is now a vector.